report/data_tasks/report/create_from_scratch/02_hubspot_read/02_data_import/01_hubspot_import.py

```diff
@@ -21,15 +21,17 @@
     Field("current_deal__services_start", "current_deal_services_start"),
     Field("current_deal__services_end", "current_deal_services_end"),
     Field("current_deal__amount", "current_deal_amount", mapping=float),
-    Field("current_deal__users_contracted", "current_deal_users_contracted"),
+    Field(
+        "current_deal__users_contracted", "current_deal_users_contracted", mapping=int
+    ),
```

Comment:
This was the field that caused the trouble, but I've added more types to other fields too